flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
disappointing

Reply:
We are dependent on JDK 11 (we can't do a newer version yet) and OpenJFX which only has x86_64 builds for JDK 11.

Reply:
there is an aarchh64 build of jdk 11

Reply:
We also have native code embedded in our Java code which doesn't have support for aarch64.  I have tried, but I haven't been able to get the native code included to build for aarch64. I am developing on a MacStudio with Apple Silicon M1 Max. I would love to support aarch64.

Reply:
The above should be JDK 17 and OpenJFX (not JDK 11). My mistake.

https://gluonhq.com/products/javafx/
When selecting JavaFX 17.0.12 [LTS], Linux, and aarch64, the only download link is for Javadoc.

And we still have our own native library code that doesn't work on aarch64. I will schedule time in October to attempt porting that native code to aarch64 again.